From 161968cdf684dd9ab79701304e97998818c96ad1 Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Thu, 6 Sep 2018 10:15:14 +0800
Subject: [PATCH] arm: dts: rk3128x: Change cpu opp-microvolt max voltage to
 1.2v

The bootloader voltage is greater than the max voltage set by
opp-microvolt. This resulting in:

[    1.949575] cpu cpu0: scale_rate=1008000000
[    1.949822] vdd_arm: Restricting voltage, 1200000-1175000uV
[    1.949854] vdd_arm: Restricting voltage, 1200000-1175000uV
[    1.949874] cpu cpu0: _set_opp_voltage: failed to set voltage (1000000 1000000 1175000 mV): -22
[    1.949889] cpu cpu0: failed to set volt 1000000

Change-Id: Id9bd6f3c7f4bf8ad2bd8a1a798ea4f8eed6b18b2
Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
---
 arch/arm/boot/dts/rk3128x.dtsi | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/arch/arm/boot/dts/rk3128x.dtsi b/arch/arm/boot/dts/rk3128x.dtsi
index 5b32f875b8e3..c411d69f4fe3 100644
--- a/arch/arm/boot/dts/rk3128x.dtsi
+++ b/arch/arm/boot/dts/rk3128x.dtsi
@@ -90,35 +90,35 @@
 
 		opp-408000000 {
 			opp-hz = /bits/ 64 <408000000>;
-			opp-microvolt = <950000 950000 1175000>;
-			opp-microvolt-L0 = <950000 950000 1175000>;
-			opp-microvolt-L1 = <950000 950000 1175000>;
+			opp-microvolt = <950000 950000 1200000>;
+			opp-microvolt-L0 = <950000 950000 1200000>;
+			opp-microvolt-L1 = <950000 950000 1200000>;
 			clock-latency-ns = <40000>;
 			opp-suspend;
 		};
 		opp-600000000 {
 			opp-hz = /bits/ 64 <600000000>;
-			opp-microvolt = <975000 975000 1175000>;
-			opp-microvolt-L0 = <975000 975000 1175000>;
-			opp-microvolt-L1 = <975000 975000 1175000>;
+			opp-microvolt = <975000 975000 1200000>;
+			opp-microvolt-L0 = <975000 975000 1200000>;
+			opp-microvolt-L1 = <975000 975000 1200000>;
 		};
 		opp-816000000 {
 			opp-hz = /bits/ 64 <816000000>;
-			opp-microvolt = <1000000 1000000 1175000>;
-			opp-microvolt-L0 = <1000000 1000000 1175000>;
-			opp-microvolt-L1 = <1000000 1000000 1175000>;
+			opp-microvolt = <1000000 1000000 1200000>;
+			opp-microvolt-L0 = <1000000 1000000 1200000>;
+			opp-microvolt-L1 = <1000000 1000000 1200000>;
 		};
 		opp-1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
-			opp-microvolt = <1175000 1175000 1175000>;
-			opp-microvolt-L0 = <1175000 1175000 1175000>;
-			opp-microvolt-L1 = <1125000 1125000 1175000>;
+			opp-microvolt = <1175000 1175000 1200000>;
+			opp-microvolt-L0 = <1175000 1175000 1200000>;
+			opp-microvolt-L1 = <1125000 1125000 1200000>;
 		};
 		opp-1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
-			opp-microvolt = <1175000 1175000 1175000>;
-			opp-microvolt-L0 = <1175000 1175000 1175000>;
-			opp-microvolt-L1 = <1125000 1125000 1175000>;
+			opp-microvolt = <1175000 1175000 1200000>;
+			opp-microvolt-L0 = <1175000 1175000 1200000>;
+			opp-microvolt-L1 = <1125000 1125000 1200000>;
 		};
 	};
 
-- 
2.35.3

